docs/docs/60-new-docs/20-quickstart/index.md

```diff
@@ -312,6 +312,58 @@ the previous section.
           discoveryLimit: 5
     ---
     apiVersion: kargo.akuity.io/v1alpha1
+    kind: PromotionTask
+    metadata:
+      name: promo-process
```

Comment:
Maybe we can be more descriptive here? E.g. `argocd-app-promotion`.

Reply:
Hmm... rethinking this. I don't want to _over-emphasize_ the Argo CD bits of this. I also agree `promo-process` seems inadequately descriptive for cases where a Project might have several different ones -- of course this one does not. I'm thinking something like "demo-promo-process" might hint at the utility of more descriptive names without overemphasizing the Argo CD bit of this.

Reply:
Fixed in ade6b4554b8660ef7b4a4b60dbc8230efda5f660